examples/http-tinygo-router/README.md

```diff
@@ -0,0 +1,74 @@
+# Spin HTTP components in TinyGo
+
+This example showcases how to build Spin HTTP components using TinyGo.
+
+```go
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	spinhttp "github.com/fermyon/spin/sdk/go/http"
+)
+
+func init() {
+	spinhttp.Handle(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "text/plain")
+		fmt.Fprintln(w, "Hello Fermyon!")
+	})
+}
+
+func main() {}
+```
+
+> For more information and examples for using TinyGo with WebAssembly, check
+> [the official TinyGo documentation](https://tinygo.org/docs/guides/webassembly/)
+> and
+> [the Wasm examples](https://github.com/tinygo-org/tinygo/tree/release/src/examples/wasm).
+
+Building this as a WebAssembly module can be done using the `tinygo` compiler:
+
+```shell
+$ spin build
+Executing the build command for component tinygo-hello: tinygo build -target=wasi -gc=leaking -no-debug -o main.wasm main.go
+Successfully ran the build command for the Spin components.
+```
+
+Finally, we can create a Spin application configuration to execute this
+component:
+
+```toml
+apiVersion = "0.1.0"
```

Comment:
```suggestion
spin_manifest_version = "1"
```

Reply:
That's the readme of the example, I'd like to update all of them in one go.